src/azure_devtools/scenario_tests/preparers.py

```diff
@@ -129,6 +129,20 @@ def process_response(self, response):
         return response
 
 
+# Function wise, enabling large payload recording has nothing to do with resource preparers
+# We still base on it so that this decorator can chain with other preparers w/o too much hacks
+class AllowLargeResponse(AbstractPreparer):
+    def __init__(self, size_kb=1024):
+        self.size_kb = size_kb
+        super(AllowLargeResponse, self).__init__('nanana', 20)
```

Comment:
nanana?

Reply:
3 n/a to make the AbstractPreparer happy on the random name length. :) . it is not really used, but a workaround to get in the decorator's chain; otherwise I need to clone out the [logics](https://github.com/Azure/azure-python-devtools/blob/master/src/azure_devtools/scenario_tests/preparers.py#L48-L58), which is not good looking either way; but I can go ahead to make it work by itself. let me know

Reply:
ok :)